app/Resources/views/admin/blog/edit.html.twig

```diff
@@ -19,7 +19,8 @@
             form: delete_form,
             button_label: 'action.delete_post'|trans,
             button_css: 'btn btn-lg btn-block btn-danger',
-            include_back_to_home_link: false
+            include_back_to_home_link: false,
+            confirmation_message: 'action.delete_post_confirmation'|trans,
```

Comment:
My concern with this feature is that we are using one variable (`confirmation_message `) for two different purposes: 1) enabling/disabling this feature, 2) defining the message to display if enabled.

Personally, multi-purpose variables makes me very nervous :cold_sweat:

Could we instead use a boolean `show_confirmation_message` option and then use the `'action.delete_post_confirmation'|trans` where needed to display the actual message contents?